src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,15 @@
+package duke;
```

Comment:
Consider adding other exceptions for handling other cases: EG: DukeWrongCommandExeption, DukeWrongParameterException, DukeInvaliddateException etc. which extend from the DukeException class.